gdmt/README.md

```diff
@@ -0,0 +1,7 @@
+# Generic Dataset Metadata Template
+
+The name `gdmt` stands for Generic Dataset Metadata Template.  For more information please see the GitHub repository at https://github.com/bmir-radx/gdmt.  This name is primarily used as a prefix for the Generic Dataset Metadata Template vocabulary.  This vocabulary can be found on BioPortal at https://bioportal.bioontology.org/ontologies/GDMT.
+
+## Contact
+
+For questions relating to this name please contact the current maintainer Matthew Horridge (matthew.horridge@stanford.edu).
```

Comment:
Can you add your github id here.  Makes it easier to check when processing updates.

Reply:
Thanks for looking at this @davidlehn.  I've made the suggested changes.